robottelo/constants.py

```diff
@@ -1642,14 +1642,14 @@
         'name': 'Subscriptions', 'controller': 'katello_subscriptions',
         'skip_for_ui': True,
     },
-    {'name': 'Products', 'controller': 'katello_products'},
+    {'name': 'Product', 'controller': 'katello_products'},
     {
         'name': 'Repository', 'controller': 'katello_repositories',
         'skip_for_ui': True
     },
-    {'name': 'GPGKey', 'controller': 'katello_gpg_keys'},
+    {'name': 'ContentCredential', 'controller': 'katello_gpg_keys', 'skip_for_ui': True},
```

Comment:
why skip?

Reply:
Leftover from debugging. As far as i remember, it wasn't working at all. Will have to submit a BZ and update True with bz id. Thanks for catching.